src/commands/tick-register.js

```diff
@@ -30,9 +29,8 @@ function register(yargs) {
     if (err)
       throw err
     
-
     server.register(user)
-    .then(user => setConfig('remote', user.couch.url))
+    .then(user => account.setUser(user))
```

Comment:
We can do that, but I'm not sure what useful information that would provide to the user. I imagine that, for the average use case, they likely wouldn't know what that is added for or what they should do with the information that it affected N documents. Thoughts?